aptos-move/move-examples/dao/nft_dao/Move.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "NFTDAO"
+version = "0.0.0"
+
+[addresses]
+dao_platform = "_"
+aptos_framework = "0x1"
+
+[dependencies]
+AptosFramework = { local = "../../../framework/aptos-framework" }
+AptosToken = { local = "../../../framework/aptos-token" }
+DataStructure = { local = "../../data_structures" }
```

Comment:
are we using this?

Reply:
yes, for buckettable

Reply:
copied the bucket_table module and deleted the dependency here